src/main/java/net/sf/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -30,9 +30,12 @@ public void readJournalListFromResource(String resourceFileName) {
     }
 
     public void readJournalListFromFile(File file) throws FileNotFoundException {
-        readJournalList(new FileReader(Objects.requireNonNull(file)));
+        try(FileReader fr = new FileReader(Objects.requireNonNull(file))) {
+            readJournalList(fr);
+        } catch(IOException e) {
+            LOGGER.info("Could not read journal list from file " + file.getAbsolutePath());
+        }
```

Comment:
Ahh, OK! Will fix that. I'll also include some more things in this PR, basically removed duplicate code from two files and moved to Util (which I know is a mess, but still slightly better than having duplicated code in lots of places...).